scripts/install-hooks.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
+HOOK_DIR=".git/hooks"
+SCRIPTS_DIR="scripts/"
+
+mkdir -p $HOOK_DIR
+mkdir -p $SCRIPTS_DIR
+
+cat > "$SCRIPTS_DIR/pre-push" << 'EOL'
+#!/bin/bash
+
+set -e
+
+echo "Running pre-push hook to check documentation..."
+
+if [[ -f "osdctl_commands.md" ]]; then
+    PRE_MD5=$(md5sum "osdctl_commands.md" | cut -d ' ' -f 1)
+else
+    PRE_MD5=""
+fi
+
+echo "Generating documentation..."
```

Comment:
I would not generate the documentation as part of installing the hooks, just use this to install the hooks.